### ISSUE TITLE
[BUG] `dart analyze` command doesn't use the plugin

### ISSUE THREAD

**Issue by DetachHead** [dart-code-checker/dart-code-metrics#757]
* **Dart code metrics version:** 4.13.0
* **Dart sdk version:** 2.16.1

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```yaml
analyzer:
  plugins:
    - dart_code_metrics

dart_code_metrics:
  rules:
    - no-empty-block
```

</details>

**What did you do? Please include the source code example causing the issue.**

```dart
void foo() {}
```

**What did you expect to happen?**
`no-empty-block` error

**What actually happened?**
```
>dart analyze lib/a.dart
Analyzing a.dart...
No issues found!
```

however it seems to work in intellij, just not when running from the cli:
![image](https://user-images.githubusercontent.com/57028336/159721177-17c0ecd3-f6dc-47fe-9c1a-f7f76572df6b.png)

do analyzer plugins only get used by IDEs and are ignored when running from the cli? this isn't the end of the world as i can still use `dart run dart_code_metrics:metrics analyze` but it would be way more convenient if i could just use `dart analyze`
